Doc updates for log aggregration and New Relic Logs UI (#7604)
magento/devdocs: src/cloud/project/new-relic.md

  - Services
---

> {{ site.data.var.ece }} subscriptions include access to New Relic services to help manage, monitor, and troubleshoot your applications and infrastructure by collecting, viewing, and analyzing data from your {{ site.data.var.ece }} project.

Reviewer: I hesitate to use "subscription". New Relic uses "subscription" with a different meaning. I think we use "plan" in the docs, **but** I would use "account" here. When I view New Relic, I have multiple projects. I have to select a Magento Account when I want to see what's going on in my instance.

Author: Changed references to projects for consistency and to avoid confusion with New Relic account.

> {{ site.data.var.ece }} subscriptions include access to New Relic services to help manage, monitor, and troubleshoot your applications and infrastructure by collecting, viewing, and analyzing data from your {{ site.data.var.ece }} project.

Author: ```suggestion
{{ site.data.var.ece }} accounts include access to New Relic services to help manage, monitor, and troubleshoot your applications and infrastructure by collecting, viewing, and analyzing data from your {{ site.data.var.ece }} project.
```

## New Relic Infrastructure

{{site.data.var.ece}} Pro projects include the [New Relic Infrastructure (NRI)][New Relic infrastructure] service, which automatically connects with the application data and performance analytics to provide dynamic server monitoring. This service is available only on Production environments.

{:.bs-callout-info}
For Pro accounts, if New Relic APM is not installed on the Staging and Production environments or New Relic Infrastructure is not available in the Production environment, submit a [support ticket]({{ site.baseurl }}/cloud/trouble/trouble.html) to request installation.

## New Relic Logs

{{site.data.var.ece}} Pro projects include the [New Relic Logs][] service on both Production and Staging environments. The service is pre-configured to aggregate all log data from your Staging and Production environments and display it in a centralized log management dashboard.

The aggregated data includes information from the following logs:

-  All {{site.data.var.ct}} and Magento application logs from the `~/var/log` directory

> -  Logs for cloud services from the `var/log/platform/<project ID>` directory

Reviewer: This is a small nit. I try to avoid using space in variables. I think it looks better and editor apps don't fuss over it as much.

```suggestion
-  Logs for cloud services from the `var/log/platform/<project-ID>` directory
```

Reviewer 2: Should be `/var/log/platform/` etc

-  Fastly CDN and WAF logs

As soon as your project is connected to New Relic, you can use the New Relic Logs service to complete tasks like the following:

-  Use [New Relic queries][New Relic query syntax for logs] to search aggregated log data
-  Visualize log data through the New Relic UI

> -  Create customer charts, dashboards, and alerts

Reviewer 2: I think you meant custom, not customer.

   magento-cloud variable:delete php:newrelic.license
   ```

> ## Using New Relic

Reviewer: The previous level 2 headings were Set up and Connect. Maybe this should be something similar? Use?

> ## Using New Relic

Author: ```suggestion
## Use New Relic
```

{:.bs-callout-info}
You can learn more about using the New Relic APM and Infrastructure agents to collect and analyze application data from the [New Relic APM] and [New Relic Infrastructure][New Relic infrastructure] documentation.

### View and analyze log data

> After you log in to your New Relic account, you can use the New Relic Logs UI to search across the aggregated log data to troubleshoot application, infrastructure, CDN, and WAF errors. Also, you can connect the log data with other data collected by New Relic APM and Infrastructure services to create charts, dashboards, and alerts to manage Magento application and cloud service operations.

Reviewer: It starts with "After you log in", but in the flow of this topic, you are in the Use/Using part already. Just an observation on flow. Also, New Relic calls Logs an app.

```suggestion
You can use the New Relic Logs application to search across the aggregated log data and troubleshoot application, infrastructure, CDN, and WAF errors. Also, you can connect the log data with other data collected by New Relic APM and Infrastructure services to create charts, dashboards, and alerts to manage Magento application and cloud service operations.
```

{:.procedure}

> To use the New Relic Logs UI:

Reviewer: application?

> To use the New Relic Logs UI:

Author: ```suggestion
To use the New Relic Logs application:
```

1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].

> 1. From the New Relic ONE home page, select **Logs**.

Reviewer 2: You may want to reverse these points, I do not believe our customers land on the New Relic ONE home page.  I believe the customer work flow is:
1. Auth
1. Select application
 > Lands them on APM page

   ![Cloud project New Relic ONE Logs service]({{ site.baseurl }}/common/images/cloud/cloud-new-relic-one-logs-access.png){:width="650px"}

   If the **Logs** application does not display, you can also access New Relic Logs using search, or navigating from the [APM applications page](#investigate-performance) navigation menu.

> 1. To review log data for cloud services, enter the `has:"ident"` query in the query field. Then, click **Query logs**.

Reviewer: This reads oddly because of the query statement. Fast readers will not see the subtle font change, maybe. Could try:

```suggestion
1. To review log data for cloud services, enter a query statement in the _Find logs where_ field. Then, click **Query logs**.
```

Reviewer 2: I like the font clarification suggestion...perhaps they can be merged so that the user doesn't have to try and re-type from the image?  Add:
 Such as `has:"ident"` somewhere to @hguthrie 's suggestion

   ![Cloud project New Relic service log data]({{ site.baseurl }}/common/images/cloud/cloud_new-relic-log-query-has-ident.png){:width="650px"}

> 1. To review Fastly log data, enter the `has: "client_ip"` query in the query field. Then, click **Query logs**.

Reviewer: ```suggestion
1. To review Fastly log data, enter the `has: "client_ip"` query in the _Find logs where_ field. Then, click **Query logs**.
```

> 1. To review Fastly log data, enter the `has: "client_ip"` query in the _Find logs where_ field. Then, click **Query logs**.

Reviewer: This is still difficult to see. It looks like it says "enter the has"
